internal/nix/shell.go

```diff
@@ -38,7 +38,7 @@ const (
 type Shell struct {
 	name            name
 	binPath         string
-	configDir       string // path to where devbox.json config resides
+	projectDir      string // path to where devbox.json config resides
 	pkgConfigDir    string
```

Comment:
I had thought about it, but then came to the conclusion that `pkgConfigDir` is the more appropriate name, since it contains the configuration for that devbox-package. WDYT?